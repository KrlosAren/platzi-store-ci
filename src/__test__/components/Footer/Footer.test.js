import { mount } from 'enzyme';
import React from 'react';
import Footer from '../../../components/Footer';

describe('Pruebas sobre el componente Footer', () => {
  const wrapper = mount(<Footer />);

  test('should be mach to snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

  test('should be show correctly', () => {
    expect(wrapper.length).toEqual(1);
  });

  test('render del titulo', () => {
    expect(wrapper.find('.Footer-title').text()).toEqual('Platzi Store!!');
  });
});

// describe('footer shapshot', () => {
//   test('comprobar la ui del componente footer', () => {
//     const footer = create(<Footer />);

//     expect(footer.toJSON()).toMatchSnapshot();
//   });
// });
