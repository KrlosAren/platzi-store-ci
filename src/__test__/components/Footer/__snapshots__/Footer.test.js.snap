// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas sobre el componente Footer should be mach to snapshot 1`] = `
<div
  className="Footer"
>
  <p
    className="Footer-title"
  >
    Platzi Store
  </p>
  <p
    className="Footer-copy"
  >
    Todos los Izquierdos Reservados
  </p>
</div>
`;
